src/components/Layout/Layout.tsx

```diff
@@ -263,17 +292,15 @@ export const Layout = ({
                     <div className="layout-sidebar-platform">
                       <PlatformNavigator
                         currentPlatform={
-                          PLATFORM_DISPLAY_NAMES[currentPlatform]
+                          currentPlatform
                         }
+                        isPrev={isPrev}
                       />
                     </div>
                   )}
 
                   <div className="layout-sidebar-menu">
-                    <Menu
-                      rootMenuNode={rootMenuNode}
-                      currentPlatform={isGen2 ? undefined : currentPlatform}
-                    />
```

Comment:
Could we combine the `gen2` and `[platform]` logic above when you're defining the `menu` variable to something like here since we don't need the `menuTitle` and `menuHref` anymore?